packages/golang/package.json

```
@@ -34,6 +34,26 @@
     "prettier": "^2.0.0",
     "typescript": "^4.0.0"
   },
+  "@rnx-kit/golang": {
```

```suggestion
  "golang-distribution": {
```

or maybe golang-release?